drivers/soundwire/intel.h

```diff
@@ -48,11 +48,23 @@ struct sdw_intel_link_res {
 	struct hdac_bus *hbus;
 };
 
+struct sdw_intel_bpt {
```

Comment:
do you want to add a description for this structure?

Reply:
Yes.